webextensions/api/extensionTypes.json

```diff
@@ -17,10 +17,12 @@
                 "notes": "This feature is supported but not exposed through the 'extensionTypes' object."
               },
               "firefox": {
-                "version_added": "45"
+                "version_added": "45",
+                "notes": "The `rect` and `scale` options were added in Firefox 82."
```

Comment:
There are no hard rules on whether to put in a note or in a separate row, but in this case putting `rect` and `scale` in a separate row would probably be helpful. See here for an example of a "type" where properties are listed separately:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/cookies/Cookie#Browser_compatibility